src/txmempool.h

```diff
@@ -602,7 +602,7 @@ class CTxMemPool
 
     /** Create a new CTxMemPool.
      */
-    explicit CTxMemPool(CBlockPolicyEstimator* estimator = nullptr);
+    explicit CTxMemPool(CBlockPolicyEstimator* estimator = nullptr, uint32_t checkFrequency = 0);
```

Comment:
Local variables and arguments names should use camel_case. See our developer notes for all our project style guidelines: https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md